packages/@aws-cdk/alexa-ask/lib/pipeline-actions.ts

```diff
@@ -0,0 +1,69 @@
+import codepipeline = require('@aws-cdk/aws-codepipeline-api');
+import cdk = require('@aws-cdk/cdk');
+
+/**
+ * Construction properties of the {@link AlexaSkillDeployAction Alexa deploy Action}.
+ */
+export interface AlexaSkillDeployActionProps extends codepipeline.CommonActionProps,
+    codepipeline.CommonActionConstructProps {
+
+  /**
+   * The client id of the developer console token
+   */
+  clientId: cdk.Secret;
+
+  /**
+   * The client secret of the developer console token
+   */
+  clientSecret: cdk.Secret;
+
+  /**
+   * The refresh token of the developer console token
+   */
+  refreshToken: cdk.Secret;
+
+  /**
+   * The Alexa skill id
+   */
+  skillId: string;
+
+  /**
+   * The source artifact containing the voice model and skill manifest
+   */
+  inputArtifact: codepipeline.Artifact;
+
+  /**
+   * An optional artifact containing overrides for the skill manifest
+   */
+  overrideArtifact?: codepipeline.Artifact;
```

Comment:
I'm not in love with this name. How about something like `skillManifestArtifact`?

Reply:
I chose the name because Alexa calls these parameters overrides. It is not the Skill manifest but a JSON containing overrides for the manifest provided as inputArtifact. Normally this is an output of CFN with the Lambda ARN.

Reply:
OK. Can you change this to `parameterOverridesArtifact` then? Thanks!